monero/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1.1
+id: monero
+category: Finance
+name: Monero Node
+icon: https://raw.githubusercontent.com/deverickapollo/umbrel-monero/dev/ui/src/assets/community-monero-icon.png
+version: "0.18.2.2"
+tagline: Run your monero node and independently store and validate every single Monero transaction with it.
+description: >-
+  Run your monero node and independently store and validate every single Monero transaction with it.
+  This is a full node that will download the entire Monero blockchain and store it on your Umbrel. 
+  This is the most secure way to run a Monero node, but it will take a long time to sync and will require a lot of storage space.
+developer: Monero
+website: https://www.getmonero.org/
+dependencies: []
+repo: https://github.com/deverickapollo/umbrel-monero
+support: https://github.com/deverickapollo/umbrel-monero/issues
+port: 9976
+submitter: Deverick
+submission: https://github.com/getumbrel/umbrel-apps/pull/690
+gallery:
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Monero%20Dashboard%20Menu.png
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Connect%20Wallet.png
+  - https://raw.githubusercontent.com/deverickapollo/umbrel-monero/master/pictures/Advanced%20Settings.png
```

Comment:
Since this will be on the official app store, these gallery images will be hosted here: https://github.com/getumbrel/umbrel-apps-gallery

This section can be changed to simple numbered images. See bitcoin for example:

https://github.com/getumbrel/umbrel-apps/blob/5108c2641ed0164dc346b0e6032a09ee2aecdf09/bitcoin/umbrel-app.yml#L29-L32